Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -2448,6 +2460,9 @@ void MediaPlayerPrivateGStreamer::handleDecryptionError(const GstStructure* stru
     fprintf(stderr, "HTML5 video: Playback failed: Decryption error [%s]\n", m_url.string().utf8().data());
     loadingFailed(MediaPlayer::FormatError);
     m_player->decryptErrorEncountered(); // override the error code
+
+    m_odhReporter.report(ODH_REPORT_AVPIPELINE_STATE_DECRYPT_ERROR, m_errorMessage.utf8().data(), OdhMediaType::VIDEO, m_avContextGetter);
```

Comment:
we have some testcase which is going to this function?

Reply:
don't know

Reply:
In that case we send both reports. I will check but I think there is no way to distinguish the error source (audio or video). Maybe base on error message?